src/Alert/Alert.levers.js

```diff
@@ -1,4 +1,4 @@
-const classnames = require('classnames')
+import clsx from 'clsx';
```

Comment:
Are we using `clsxd` for everything or just as needed?

Reply:
I would say just as needed. No sense on adding cpu cycles.

Reply:
Probably can remove those from the levers.

Reply:
Thank you for that whole bunch of revisions!